third_party/DMP/diff_match_patch_uncompressed.js

```diff
@@ -26,7 +26,7 @@
  * Class containing the diff, match and patch methods.
  * @constructor
  */
-function diff_match_patch() {
+var diff_match_patch = function() {
```

Comment:
If this is a class, shouldn't it be called DiffMatchPatch?  (I am aware that it is not feasible to change it now…)

Reply:
Yes, the name was chosen pre-Google.  Stuck with it now.